### PR TITLE
Create nas-sc03ze.xml

### PR DESCRIPTION
**Pull request by jjnh**
Added support for ShenZhen Neo Coolcam NAS-SC03ZE roller shutter switch

### CODE REVIEW

**Comment by jjnh** [comment]
My apologies I am not a developer and this is very new for me. I checked the link to the picture and apparently the page had gone offline. I found a new picture link and link to the manual.
The configuration files I added have been running without fault for almost a month in my configuration (RPi/Domoticz/OpenZwave).
Can someone help me?

File: config/shenzen_neo/nas-sc03ze.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+					Button Switch
+				</Help>
+		  <Item value="0" label="Button Switch Disabled"/>
+		  <Item value="1" label="Button Switch Enabled"/>
+		</Value>
+	</CommandClass>
+  <!-- Association Groups -->
+<CommandClass id="133">
+		<Associations num_groups="1">
+		  <Group index="1" label="Lifeline" max_associations="5" auto="true"/>
```
Comment (firstof9) [inline config/shenzen_neo/nas-sc03ze.xml:71]: Remove the `auto="true"` parts here.

**Review by firstof9 — CHANGES_REQUESTED** [review]

**Comment by firstof9** [comment]
Also bump the revision number in manufacturer_specific.xml

**Comment by jjnh** [comment]
Thanks very much for you help. I have removed both the Auto and the revision number in the current patch-2.
Like I said I am not a developer so, how do I continue now?

File: config/manufacturer_specific.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ManufacturerSpecificData Revision="133" xmlns="https://github.com/OpenZWave/open-zwave">
+<ManufacturerSpecificData xmlns="https://github.com/OpenZWave/open-zwave">
```
Comment (firstof9) [inline config/manufacturer_specific.xml:2]: `Revision="134"` needs to be placed in here.

**Review by firstof9 — CHANGES_REQUESTED** [review]

File: config/manufacturer_specific.xml (hunk 2 above)
Comment (jjnh) [reply config/manufacturer_specific.xml]: Done

Thanks again for the support!

**Comment by jjnh** [comment]
Thanks very much. I will delete them.

**Pull request merged**